.goreleaser.yml

```diff
@@ -1,5 +1,6 @@
 project_name: fastly
 release:
+  draft: true
```

Comment:
This will cause the release to be a draft, and thus allow me to manually copy/paste the relevant CHANGELOG content into it before publishing. This is how the Terraform repo works.